monitoring/types.go

```diff
@@ -0,0 +1,22 @@
+package monitoring
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	longhornName = "longhorn"
+
+	subsystemVolume      = "volume"
+	subsystemBackupStats = "backup_stats"
+	subsystemNodeStats   = "node_stats"
+
+	prometheusNodeIDLabel    = "node_id"
```

Comment:
remove the `id` suffix and just use `node, volume, condition, condition_reason`

Reply:
Thank you! Fixed